tests/integration/components/google-recaptcha-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | google-recaptcha', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`{{google-recaptcha}}`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      {{#google-recaptcha}}
+        template block text
+      {{/google-recaptcha}}
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
```

Comment:
need some tests. use `sinon` to mock out the window methods that the google script provides and ensure the callbacks run in the expected order.